.gitignore

```diff
@@ -15,3 +15,6 @@ TODO.md
 pkg/columnstore/testdata/merge-bench
 
 data/
+
+# IDE
+.vscode/
```

Comment:
Please add this to a global gitignore on your system rather than to this repository. 
https://stackoverflow.com/questions/7335420/global-git-ignore

Reply:
ack, this is excluded now